staff/aleix-caminal/js-arrays/concat.js

```diff
@@ -0,0 +1,16 @@
+document.querySelector('#concat').addEventListener('click', function() {
+    var arr = [1, 2, 3];
+    var arr2 = [4, 5, 6];
+    var res = concat(arr, arr2)
+    console.log(res);
+});
+
+function concat(...elems) {
```

Comment:
you are using the spread operator... does this code work without it? I believe spread operator is ES6 and we are not allowed to use it!